projects/angular/clarity.api.md

```diff
@@ -318,9 +318,9 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
     // (undocumented)
     panelOpenChange: EventEmitter<boolean>;
     // (undocumented)
-    stepCompleteText(panelNumber: number): string;
+    protected stepCompleteText(panelNumber: number): string;

```

Comment:
@kevinbuhmann  they still do show up in the public api

Reply:
Okay, I misremembered something then. Sorry about that. In any case, these don't need to to be `public` anyway.